src/util/config.js

```diff
@@ -30,7 +30,15 @@ const buildConfig = () => {
       case 'Measure':
         config.profiles['Measure'] = {
           service: path.resolve('src', 'services', 'measure.service.js'),
-          versions: [VERSIONS['4_0_0']]
+          versions: [VERSIONS['4_0_0']],
+          operation: [
+            {
+              name: 'submitData',
+              route: '/$submit-data',
+              method: 'POST',
+              reference: 'http://hl7.org/fhir/OperationDefinition/Measure-submit-data'
+            }
```

Comment:
should add another custom operation that allows a measure ID to be passed in as an arg in the URL

Reply:
Yes. Per the [spec](https://www.hl7.org/fhir/measure-operation-submit-data.html), it should be supported on both `Measure/$submit-data` and `Measure/[id]/$submit-data`

Hopefully, the same function will invoke in both cases, the args passed into the function will just be different? Worth checking.